website/content/api-docs/agent/check.mdx

```diff
@@ -188,6 +188,11 @@ The table below shows this endpoint's support for
   The value can be further limited for all checks of a given agent using the
   `check_output_max_size` flag in the agent.
 
+- `TLSServerName` `(string: "")` - Specifies an optional string used to set the
```

Comment:
These docs looks good. I think we also need to document this in `website/content/docs/discovery/checks.mdx` , which is linked from the config docs.